src/IRac.cpp

```diff
@@ -285,16 +285,8 @@ bool IRac::isProtocolSupported(const decode_type_t protocol) {
 #if SEND_VOLTAS
     case decode_type_t::VOLTAS:
 #endif
-#if SEND_WHIRLPOOL_AC
```

Comment:
Always enabled?

Reply:
Yep. It sucks. I ran out of nicer hacks to get it to ignore the compiler warning/error when every protocol in the `switch` statement is disabled. I.e. the `return` statement is never reached.

If you have a better solution, I'm all ears. See commits fire some of the things I tried. I tried way more.

Reply:
@NiKiZe  In short (as I'm not a mobile device answering this now), It's a horrible hack (hardwiring at least one protocol to always be there) to shut up the compiler warning when NO protocols have been enabled, thus having an unreachable code warning/error. It's not a lie, the protocol is a supported protocol by `IRac`, and it won't break anything, but it's been disabled.
The other _hack_ option I had was to _always_ include **all** the supported IRac protocols, to be _equal_. Hence, this was the _least worst_ option, I felt.